pyfar/plot/plotstyles/light.mplstyle

```diff
@@ -20,8 +20,9 @@ font.style          : normal
 font.variant        : normal
 font.weight         : normal
 font.stretch        : normal
-font.size           : 10.0
```

Comment:
I would not specify font sizes, they are resolution/display specific

Reply:
Agreed and removed all font size specs from both plot styles...